### PR TITLE
Fix feature flag schema support

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

Add support to the json validator for the feature flags.
In `create-validation-function.ts` the feature flags are now loaded as well and right before a schema
gets parsed the specified metric names as found in `features.ts` are removed from the `required`
lists.
That way the validator will consider these metrics optional, yet during the `generate-typescript` phase
they are required. This way we can disable a feature while at the same keeping the compiler happy
without having to add all sorts of undefined checks everywhere in our code.

## Motivation

Better DX, hopefully

### Governance

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: packages/cli/src/schema/create-validate-function.ts [hunk 1]
```diff
@@ -1,36 +1,82 @@
...
+      disableFeatureFlagMetrics(schema, disabledMetrics);
+    }
+    return schema;
   } catch (e) {
     if (e instanceof Error)
       throw new Error(`Error while parsing file ${schemaPath}:\n${e.message}`);
   }
 }
 
+function disableFeatureFlagMetrics(schema: any, features: Feature[]) {
```
Comment (VWSCoronaDashboard17) [inline packages/cli/src/schema/create-validate-function.ts:30]: Should it maybe check if the feature flag `isEnabled`?

Comment (VWSCoronaDashboard1) [reply packages/cli/src/schema/create-validate-function.ts:30]: the features are already filtered at the top of the page where they are imported:
`const disabledMetrics = features.filter((x) => !x.isEnabled);`
So I only disable the metrics for the features that have `isEnabled` set to false.

**Comment by VWSCoronaDashboard1** [comment]
> Would it be a good idea to also merge in the changes in #3440 here as that contains an upgrade to `ajv`?

no, I'm not going to merge in a major version upgrade right before a release ;)

**Comment by VWSCoronaDashboard17** [comment]
Yep, that makes sense now that I know this needs to go into master.

**Pull request merged**